src/weather_gen.cpp

```diff
@@ -265,10 +265,11 @@ void weather_generator::test_weather() const
     const time_point begin = calendar::turn;
     const time_point end = begin + 2 * calendar::year_length();
     for( time_point i = begin; i < end; i += 200_turns ) {
-        w_point w = get_weather( tripoint_zero, to_turn<int>( i ), 1000 );
+        //@todo: a new random value for each call to get_weather? Is this really intended?
+        w_point w = get_weather( tripoint_zero, to_turn<int>( i ), rng_bits() );
         weather_type c =  get_weather_conditions( w );
         weather_datum wd = weather_data( c );
-        testfile << to_turn<int>( i ) << ";" << w.temperature << ";" << w.humidity << ";" << w.pressure <<
+        testfile << to_turn<int>( i ) << "," << w.temperature << "," << w.humidity << "," << w.pressure <<
```

Comment:
These two changes to `weather_gen.cpp` seem random and unrelated.

Reply:
Indeed.